apps/hash-graph/bin/cli/src/subcommand/type_fetcher.rs

```diff
@@ -78,7 +78,7 @@ pub async fn type_fetcher(args: TypeFetcherArgs) -> Result<(), GraphError> {
             channel.execute(server.serve())
         })
         .buffer_unordered(255)
-        .for_each(|_| async {})
+        .for_each(|()| async {})
```

Comment:
non blocking, just out of curiosity, which is the empty async block needed here?

Reply:
For iterators you would typically use `.for_each(drop)`, but I'm not aware of a similar approach for streams. We need to consume the iterator but `collect` seems wrong here (even though it's `()`).

Reply:
I changed that line btw because `clippy` complained.